server.py

```diff
@@ -27,28 +27,15 @@
 KNOWN_SURVEYS = {
     "0.0.1": {
         "009": [
```

Comment:
Why the restructure of the list layout?

Reply:
Just to make it tidier.  If I emulated this list with the new survey, this file will be really long.  I thought having the formtypes in rows of 10 seemed to make it tidier.